src/main/java/in/hp/java/userservice/controller/UserController.java

```diff
@@ -0,0 +1,46 @@
+package in.hp.java.userservice.controller;
+
+import in.hp.java.userservice.model.User;
+import in.hp.java.userservice.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class UserController {
```

Comment:
It if better practice to create interface for Controller which will contain configuration related APIs. And Class which will have only business logic. So in future if we want to add swagger configuration so we can add it in interface and our class is clean to see only business logic.

Reply:
Added interface implementation for UserController